[INFO -- 2018-10-09 12:25:50]
Starting flip_value script
version: 1.1
[INFO -- 2018-10-09 12:25:50]
Initialization strings:
[INFO -- 2018-10-09 12:25:50]
(1) file /tmp/fft_nn/fft_nn.out
[INFO -- 2018-10-09 12:25:50]
(2) set args 32768 /tmp/fft_nn/output_falha.data
[INFO -- 2018-10-09 12:25:50]
initSignal:10
[INFO -- 2018-10-09 12:25:50]
endSignal:15
[INFO -- 2018-10-09 12:25:50]
faultModel:2
[EXCEPTION -- 2018-10-09 12:26:02]
pythonException: 'bool' object is not iterable
[EXCEPTION -- 2018-10-09 12:26:02]
gdbException: Tentativo de desreferenciar um ponteiro genérico.
[INFO -- 2018-10-09 12:26:02]
Backtrace BEGIN:
#0  _IO_vfscanf_internal (s=s@entry=0x6062c0, format=<optimized out>, argptr=argptr@entry=0x7fffffffd408, errp=errp@entry=0x0) at vfscanf.c:440
#1  0x00007ffff7032162 in __isoc99_fscanf (stream=0x6062c0, format=<optimized out>) at isoc99_fscanf.c:34
#2  0x00007ffff7bcf14e in fann_create_from_fd () from /usr/local/lib/libfann.so.2
#3  0x00007ffff7bcf608 in fann_create_from_file () from /usr/local/lib/libfann.so.2
#4  0x00000000004019b7 in radix2DitCooleyTykeyFft(int, int*, Complex*, Complex*) ()
#5  0x0000000000401631 in main ()

435	vfscanf.c: Arquivo ou diretório não encontrado.

Backtrace END
Fliping array at pos: 15
Memory content before bitflip:0x65
Fault Model: Random bit-flip
base address to flip value: 0x7ffff715cc0f  
address max offset: 1
Memory content after  bitflip:0xa5
frame name: _IO_vfscanf_internal
symbol name: __PRETTY_FUNCTION__
symbol filename: vfscanf.c
symbol line: 0
value: "_IO_vfscanf_int\245rnal"
value address: 0x7ffff715cc00 <__PRETTY_FUNCTION__.11925>
Type: The type is an array.
Type sizeof: 21
Field name: None
Field Type: A range type, that is, an integer type with bounds.
Field Type sizeof: 8
Field Type range: (0, 20)

[INFO -- 2018-10-09 12:26:02]
Fault Injection Successful after 12.81188702583313s
[INFO -- 2018-10-09 12:26:10]
event type: exit
[INFO -- 2018-10-09 12:26:10]
exit code: 0
